ibis/backends/tests/test_generic.py

```diff
@@ -1758,6 +1759,74 @@ def hash_256(col):
     backend.assert_series_equal(h1, h2)
 
 
+@pytest.mark.parametrize(
+    ("from_type", "to_type", "from_val", "expected"),
+    [
+        param("int", "float", 0, 0.0),
+        param("float", "int", 0.0, 0),
+        param("string", "int", "0", 0),
+        param("string", "float", "0", 0.0),
+        param(
+            "timestamp",
+            "string",
+            datetime.datetime(2023, 1, 1),
+            "2023-01-01 00:00:00",
```

Comment:
Why is this the expected value?

Why wouldn't we include the subsecond values in the string output, for example?

I don't think this particular test is fleshed out enough to be included here, it's stipulating a specific behavior for something that needs more discussion before writing a test.

You've marked a bunch of these tests as broken, and that's not really true here. Broken should really be reserved for things that are bugs in Ibis that are ~impossible to workaround, behaviors in the backend that are ~impossible to workaround, or bugs in the backend that are ~impossible to workaround.

Here's it's just the old problem of "things behave differently".

I think these test cases should be removed, because there's no consensus on what the behavior should be other than "return a string".

Reply:
yeah this is a good point, thanks. So should we just open an issue to track that, and delete these cases? I'm fine with that.

re test labels: what should I use as the marker for "backends behave differently, it's possible for us to paper over the top of that in ibis, but we just haven't gotten to it yet"? `mark.notimpl`? If so, I can update https://ibis-project.org/contribute/02_workflow#setting-up-non-trivial-backends to match, as I interpreted that to mean that I should use `mark.broken` so I think it is a bit misleading.

Reply:
Yeah, we should probably revisit those docs a bit.

Also possible we all have slightly different interpretations of the markers so maybe we should clarify that too :)

I think these would be ... notyet?

Perhaps it's time to consider a new marker for this scenario? I'm sure @gforsyth has some thoughts here.

Reply:
`pytest.mark.welcome_to_the_jungle`?

I agree that `broken` isn't right for this. I think I added that marker initially so that contributors who opened PRs that exposed previously broken functionality weren't on the hook for fixing things way out of scope.

For these tests to go in, I think we want to first agree on what the "correct" (or just our expected) precision and format is for a timestamp.

Then, for those backends that support a custom format for timestamp outputs, those would be marked as `notimpl`, since it would be on us to add in handling for massaging the output into the desired format.

For those backends that don't offer a way to massage the output, those would be `notyet`.


The goal here, always, is to note which things require only effort from Ibis contributors (`notimpl`) vs. those which require upstream changes (`notyet`) vs. those which should be abandoned as folly (`never`).

Reply:
tracking issue for timestamp -> string casting: #9534 

tracking issue for test mark adjustments: https://github.com/ibis-project/ibis/issues/9535